### PR TITLE
Added Burner Bonanza [Account]

### LINKED ISSUE #106: burner.bonanza.com
The Background Burner quickly removes the background from any image or photo. Our patent-pending technology does all the work for you, automatically.
Free online tool automatically removes the background from images
Simply drag image to remove background.
Can this go in Graphics & Image section @aviaryan ?

### PR DESCRIPTION
**Pull request by Anshuman-Verma**
Fixes #106 

**https://burner.bonanza.com/**

**Tool to burn/remove the background of  an image and instantly share on Facebook or Twitter, login to download the edited image as PNG or JPEG.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- [x] I have read the [Contribution guidelines](https://github.com/aviaryan/awesome-no-login-web-apps/blob/master/CONTRIBUTING.md) and I am confident that my PR is suitable. 
- [x] This pull request has a descriptive title. For example, `Add [App name]`, not `Update readme.md` or `Added new entries`.
- [x] The app I added is not a duplicate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +144,7 @@ Powershell, Redis and lots others. Also has a good collection of IDEs for progra
 * [Realtime Board](https://realtimeboard.com/) `[Account]` - Collaborative blackboard with huge area, screen sharing, video & voice communication.
 * [ezGIF](http://ezgif.com/) - GIF images editor (cropping, scaling, optimizing and more).
 * [Adobe Color CC](https://color.adobe.com/) - Color scheme designer.
+* [Burner Bonanza](https://burner.bonanza.com/) - Tool to remove the background of an image.
```
Comment (aviaryan) [inline README.md]: Add more details like export formats supported, any other limitations etc

Comment (Anshuman-Verma) [inline README.md]: Updating

**Comment by Anshuman-Verma** [comment]
We need an acc to download the image, should i add [Acc] tag ?

**Comment by aviaryan** [comment]
@Anshuman-Verma Yes, then it's Account type. BTW, do we have a non-account type alternative here? It would be great if it exists.

**Comment by Anshuman-Verma** [comment]
@aviaryan i checked some 7 background removal sites, ones that provide background removal without account, the only problem in those web-apps is that we need to manually select the area to be clipped off whereas in the above mentioned web-app removal is done automagically without much hassl,e though it needs an account but is more promising. There is another similar web-app https://www.malabi.co which provides automated bg removal but needs account :(
I can remove Burner Bonanza and add apps which require manual selection of area to be conserved, do let me know what we need here.

**Comment by aviaryan** [comment]
Thank you @Anshuman-Verma for the effort. I have tried manual background removing websites and they are really hard to use and perform inconsistently. So let's keep BurnerBonanza in our list.

**Pull request merged**